src/Build/BackEnd/Shared/BuildRequestConfiguration.cs

```diff
@@ -706,7 +699,7 @@ public void RetrieveFromCache()
                     return;
                 }
 
-                ITranslator translator = GetConfigurationTranslator(TranslationDirection.ReadFromStream);
+                using ITranslator translator = GetConfigurationTranslator(TranslationDirection.ReadFromStream);
                 try
                 {
```

Comment:
safe to remove this try/finally as well?

Reply:
Definitely. Thank you!